bindings/python/test/trajectory/state/test_coordinate_subset.py

```diff
@@ -30,6 +30,9 @@ def test_eq(self, coordinate_subset: CoordinateSubset):
     def test_ne(self, coordinate_subset: CoordinateSubset):
         assert (coordinate_subset != coordinate_subset) == False
 
+    def test_hash(self, coordinate_subset: CoordinateSubset):
+        assert hash(coordinate_subset) is not None
```

Comment:
Should we pop in a couple coordinates to verify behavior?

To cover the cases of:
- Different key, value insertions can be grabbed as intended
- Different key objects but `==` and `hash` produce true and the same value, insert both and insure only the value for the second instance.